schema.graphql

```diff
@@ -156,3 +156,29 @@ type CollateralAllowedChangeEvent @entity {
   controller: PaprController!
   allowed: Boolean!
 }
+
+enum ActivityType {
+  ADD_COLLATERAL
+  REMOVE_COLLATERAL
+  INCREASE_DEBT
+  REDUCE_DEBT
+  SWAP
+  START_AUCTION
+  END_AUCTION
+  ADD_COLLATERAL_INCREASE_DEBT
+  ADD_COLLATERAL_INCREASE_DEBT_SWAP
+  REMOVE_COLLATERAL_REDUCE_DEBT
+  REMOVE_COLLATERAL_REDUCE_DEBT_SWAP
+  INCREASE_DEBT_SWAP
+  REDUCE_DEBT_SWAP
+}
+
+type Activity @entity {
+  id: ID! # tx hash of activity
+  description: String!
+  type: ActivityType!
+  timestamp: Int!
+  metadata: String!
+  vault: Vault
+  ethPrice: BigInt
```

Comment:
I think we want paprPrice in ETH? do we also want total out/in or that goes in the metadata?

Reply:
@wilsoncusack good point.

I was thinking total out/in would go in metadata